packages/messaging/README.md

```diff
@@ -94,7 +96,7 @@ import com.adobe.marketing.mobile.MobileCore;
 import com.adobe.marketing.mobile.Edge;
 import com.adobe.marketing.mobile.edge.identity.Identity;
 import com.adobe.marketing.mobile.Messaging;
-  
+
```

Comment:
Should we also match with iOS to add optimize extension here.  Btw the way, is optimize extension required for in-app to work?